docs/integrations/sources/postgres.md

```diff
@@ -1,221 +1,151 @@
-# Postgres
-
-This page contains the setup guide and reference information for the Postgres source connector for CDC and non-CDC workflows.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## When to use Postgres with CDC
-
-Configure Postgres with CDC if:
+# Postgres
 
-- You need a record of deletions
-- Your table has a primary key but doesn't have a reasonable cursor field for incremental syncing (`updated_at`). CDC allows you to sync your table incrementally
+Airbyte's certified Postgres connector offers the following features:
+* Multiple methods of keeping your data fresh, including [Change Data Capture (CDC)](./../../understanding-airbyte/cdc.md) and replication using the [xmin system column](#xmin-replication-mode). 
```

Comment:
Do you want to add recommended mode is CDC if syncing tables in the terabyte range?

Reply:
That's outlined multiple times further down in the doc. I think this section should be more on selling the dream than starting with limitations and pointing customers down certain paths.

In the quick start section, when users configure xmin, I refer them to CDC if their tables are above a certain size.